std/experimental/allocator/mallocator.d

```diff
@@ -198,7 +212,8 @@ struct AlignedMallocator
     /**
     Forwards to $(D alignedAllocate(bytes, platformAlignment)).
     */
-    @nogc @trusted void[] allocate(size_t bytes) shared
+    @trusted @nogc nothrow
```

Comment:
Well it returns `void[]`, which requires an un`@safe` cast, so maybe `@system` is more appropriate at this level. Higher level stuff like `make` or `Unique!Allocator` are more appropriate candidates for `@safe`-ty, IMO.